src/Tizen.NUI.Components/Controls/ItemsView/OneLineLinearItem.cs

```diff
@@ -0,0 +1,482 @@
+/* Copyright (c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Binding;
+using Tizen.NUI.Components.Extension;
+using Tizen.NUI.Accessibility;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// Button is one kind of common component, a button clearly describes what action will occur when the user selects it.
+    /// Button may contain text or an icon.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class OneLineLinearItem : ViewItem
```

Comment:
`[EditorBrowsable(EditorBrowsableState.Never)]`가 빠졌습니다.
```suggestion
    [EditorBrowsable(EditorBrowsableState.Never)]
    public class OneLineLinearItem : ViewItem
```